### PR TITLE
Promote sidecar shipper.upload-compacted flag to non hidden, non experimental state.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartek Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: docs/quick-tutorial.md [hunk 1]
```diff
@@ -93,6 +93,11 @@ thanos sidecar \
 
 * _[Example Kubernetes manifests using Prometheus operator](https://github.com/coreos/prometheus-operator/tree/master/example/thanos)_
 
+### Uploading old metrics.
+
+When sidecar is ran wit `--shipper.upload-compacted` flag it will sync all older existing blocks from the Prometheus local storage on startup.
```
Comment (squat) [inline docs/quick-tutorial.md]: When the sidecar is run with the `--...`

File: docs/quick-tutorial.md [hunk 2]
```diff
@@ -93,6 +93,11 @@ thanos sidecar \
 
 * _[Example Kubernetes manifests using Prometheus operator](https://github.com/coreos/prometheus-operator/tree/master/example/thanos)_
 
+### Uploading old metrics.
+
+When sidecar is ran wit `--shipper.upload-compacted` flag it will sync all older existing blocks from the Prometheus local storage on startup. 
+NOTE: This assumes you never run sidecar with block uploading against this bucket. Otherwise manual steps are needed to remove overlapping blocks from bucket.
```
Comment (squat) [inline docs/quick-tutorial.md]: ... from the bucket.

File: docs/quick-tutorial.md [hunk 3]
```diff
@@ -93,6 +93,11 @@ thanos sidecar \
...
+Those will be suggested by sidecar verification process.
```
Comment (squat) [inline docs/quick-tutorial.md]: ... by the sidecar...

**Comment by squat** [comment]
Hmm could we have a race?
```
storeset_test.go:476:

	exp: 15

	got: 14

--- FAIL: TestStoreSet_Update (4.02s)
FAIL
FAIL	github.com/thanos-io/thanos/pkg/query	8.073s
```

**Comment by bwplotka** [comment]
@squat  Will look on that. Not introduced by this PR for sure (:

**Comment by squat** [comment]
Yes that’s for sure. We’ll need to do a bit of digging to identify.

**Pull request merged**